lib/snippets/publish-lerna-independent-packages

```diff
@@ -28,10 +28,12 @@ const packages = PackageUtilities.topologicallyBatchPackages(unsortedPackages, t
   .reduce((acc, packageGroup) => [...acc, ...packageGroup], [])
   .filter(({name}) => taggedPackages.includes(name));
 
-packages.forEach(({name, version}) => {
-  const command = `node_modules/.bin/lerna publish --yes --skip-npm=false --skip-git --force-publish=${name} --repo-version=${version} --scope=${name} --npm-tag=${npmTag(
-    version,
-  )}`;
+packages.forEach(({name, version, isPrivate}) => {
+  const access = isPrivate ? 'restricted' : 'public';
```

Comment:
should I even bother with this, or should we just use the package.json config for `npm` publishing as specified here: https://development.shopify.io/guides/package_cloud_nodejs#?

IMO we should keep this, bc it seems to allow using the `private: true` or `private: false` flag in the package.json file which is more intuitive.

Reply:
The `private`, `package.json` option is very misleading. It doesn't dictate whether a package is published privately or publicly. It prevents deploys entirely - [docs](https://docs.npmjs.com/files/package.json#private)